<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP3 Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="trackList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <item>
       <widget class="QToolButton" name="prevButton">
        <property name="icon">
         <iconset>
          <normaloff>:/icons/prev.svg</normaloff>
         </iconset>
        </property>
        <property name="toolTip">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playButton">
        <property name="icon">
         <iconset>
          <normaloff>:/icons/play.svg</normaloff>
         </iconset>
        </property>
        <property name="toolTip">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pauseButton">
        <property name="icon">
         <iconset>
          <normaloff>:/icons/pause.svg</normaloff>
         </iconset>
        </property>
        <property name="toolTip">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopButton">
        <property name="icon">
         <iconset>
          <normaloff>:/icons/stop.svg</normaloff>
         </iconset>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextButton">
        <property name="icon">
         <iconset>
          <normaloff>:/icons/next.svg</normaloff>
         </iconset>
        </property>
        <property name="toolTip">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add tracks...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="positionLayout">
      <item>
       <widget class="QSlider" name="positionSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>00:00 / 00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>